-code-checks

ERROR: Build errors found during clang-tidy run.
```
RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc:159:74: error: use of undeclared identifier 'wilo'; did you mean 'wglo'? [clang-diagnostic-error]
    lp0 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wilo, wihi )).first;
                                                                         ^~~~
                                                                         wglo
--
RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc:159:80: error: use of undeclared identifier 'wihi'; did you mean 'wghi'? [clang-diagnostic-error]
    lp0 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wilo, wihi )).first;
                                                                               ^~~~
                                                                               wghi
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoLocalMuon/CSCRecHitD/src/CSCHitFromStripOnly.cc
+++ b/RecoLocalMuon/CSCRecHitD/src/CSCHitFromStripOnly.cc
@@ -471,59 +471,67 @@ void CSCHitFromStripOnly::findMaxima(const CSCDetId& id) {
       }
        
       bool additional_maxima_found = false;
-      // search for additional maxima if: 
-	  // - hit is closer than 3 strips from the edge
-	  // - enough consecutive strips with signal
-	  // - strip charge distribution looks abnormal
-      
-      if ( i > 2 && i < (thePulseHeightMap.size()-3) && numberOfConsecutiveStrips > 3){
-	//try to look for additional maxima at the left side from the main maxima
+      // search for additional maxima if:
+      // - hit is closer than 3 strips from the edge
+      // - enough consecutive strips with signal
+      // - strip charge distribution looks abnormal
 
-if(((thePulseHeightMap[i+1].phmax() >= thePulseHeightMap[i-1].phmax() &&
-	    thePulseHeightMap[i+1].phmax() >= thePulseHeightMap[i-2].phmax() &&
-	    thePulseHeightMap[i+2].phmax() <= thePulseHeightMap[i-2].phmax())|| 
-	   (thePulseHeightMap[i+1].phmax() <= thePulseHeightMap[i-1].phmax() &&
-	    thePulseHeightMap[i+1].phmax() <= thePulseHeightMap[i-2].phmax())) && 
-	   thePulseHeightMap[i-1].phmax() >= thePulseHeightMap[i-2].phmax() &&//to avoid close maxima delimitation (this is already present in the code)
-	   thePulseHeightMap[i-2].phmax() > 20){//no need in a small charge maxima (might need adjustment)
+      if (i > 2 && i < (thePulseHeightMap.size() - 3) && numberOfConsecutiveStrips > 3) {
+        //try to look for additional maxima at the left side from the main maxima
 
-	  additional_maxima_found = true;
-	  theMaxima.push_back(i-2);//insert left maxima first
-	  theConsecutiveStrips.push_back(numberOfConsecutiveStrips);//insert the same number of cosecutive strips, because they belong to both maximas
-	  theMaxima.push_back(i);//insert main maxima
-	  theConsecutiveStrips.push_back(numberOfConsecutiveStrips);//insert the same number of cosecutive strips, because they belong to both maximas
-	  
-	}//looking for additional maxima on the left
-	
-	//try to look for additional maxima at the right side from the main maxima
-if(((thePulseHeightMap[i+1].phmax() >= thePulseHeightMap[i-1].phmax() &&
-	    thePulseHeightMap[i+2].phmax() >= thePulseHeightMap[i-1].phmax()) || 
-	   (thePulseHeightMap[i+1].phmax() <= thePulseHeightMap[i-1].phmax() &&
-	    thePulseHeightMap[i+2].phmax() <= thePulseHeightMap[i-1].phmax() &&
-	    thePulseHeightMap[i+2].phmax() >= thePulseHeightMap[i-2].phmax())) && 
-	   thePulseHeightMap[i+1].phmax() >= thePulseHeightMap[i+2].phmax() &&//to avoid close maxima delimitation (this is already present in the code)
-	   thePulseHeightMap[i+2].phmax() > 20){//no need in a small charge maxima (might need adjustment)
-	
-	  additional_maxima_found = true;
-	  theMaxima.push_back(i);//insert main maxima first
-	  theConsecutiveStrips.push_back(numberOfConsecutiveStrips);//insert the same number of cosecutive strips, because they belong to both maximas
-	  theMaxima.push_back(i+2);//insert right maxima
-	  theConsecutiveStrips.push_back(numberOfConsecutiveStrips);//insert the same number of cosecutive strips, because they belong to both maximas
-	  
-	}//looking for additional maxima on the right
-    
-	//if nothing additional found fill the maxima
-	if(!additional_maxima_found){
-	  theMaxima.push_back(i);
-	  theConsecutiveStrips.push_back(numberOfConsecutiveStrips);
-	}
-  }else{//not the case for looking for the additional maxima
-    theMaxima.push_back(i);
-    theConsecutiveStrips.push_back(numberOfConsecutiveStrips);
-  }
-    }//if maximafound
-  }//all pulses
-}//find maxima procedure
+        if (((thePulseHeightMap[i + 1].phmax() >= thePulseHeightMap[i - 1].phmax() &&
+              thePulseHeightMap[i + 1].phmax() >= thePulseHeightMap[i - 2].phmax() &&
+              thePulseHeightMap[i + 2].phmax() <= thePulseHeightMap[i - 2].phmax()) ||
+             (thePulseHeightMap[i + 1].phmax() <= thePulseHeightMap[i - 1].phmax() &&
+              thePulseHeightMap[i + 1].phmax() <= thePulseHeightMap[i - 2].phmax())) &&
+            thePulseHeightMap[i - 1].phmax() >=
+                thePulseHeightMap[i - 2]
+                    .phmax() &&  //to avoid close maxima delimitation (this is already present in the code)
+            thePulseHeightMap[i - 2].phmax() > 20) {  //no need in a small charge maxima (might need adjustment)
+
+          additional_maxima_found = true;
+          theMaxima.push_back(i - 2);  //insert left maxima first
+          theConsecutiveStrips.push_back(
+              numberOfConsecutiveStrips);  //insert the same number of cosecutive strips, because they belong to both maximas
+          theMaxima.push_back(i);          //insert main maxima
+          theConsecutiveStrips.push_back(
+              numberOfConsecutiveStrips);  //insert the same number of cosecutive strips, because they belong to both maximas
+
+        }  //looking for additional maxima on the left
+
+        //try to look for additional maxima at the right side from the main maxima
+        if (((thePulseHeightMap[i + 1].phmax() >= thePulseHeightMap[i - 1].phmax() &&
+              thePulseHeightMap[i + 2].phmax() >= thePulseHeightMap[i - 1].phmax()) ||
+             (thePulseHeightMap[i + 1].phmax() <= thePulseHeightMap[i - 1].phmax() &&
+              thePulseHeightMap[i + 2].phmax() <= thePulseHeightMap[i - 1].phmax() &&
+              thePulseHeightMap[i + 2].phmax() >= thePulseHeightMap[i - 2].phmax())) &&
+            thePulseHeightMap[i + 1].phmax() >=
+                thePulseHeightMap[i + 2]
+                    .phmax() &&  //to avoid close maxima delimitation (this is already present in the code)
+            thePulseHeightMap[i + 2].phmax() > 20) {  //no need in a small charge maxima (might need adjustment)
+
+          additional_maxima_found = true;
+          theMaxima.push_back(i);  //insert main maxima first
+          theConsecutiveStrips.push_back(
+              numberOfConsecutiveStrips);  //insert the same number of cosecutive strips, because they belong to both maximas
+          theMaxima.push_back(i + 2);  //insert right maxima
+          theConsecutiveStrips.push_back(
+              numberOfConsecutiveStrips);  //insert the same number of cosecutive strips, because they belong to both maximas
+
+        }  //looking for additional maxima on the right
+
+        //if nothing additional found fill the maxima
+        if (!additional_maxima_found) {
+          theMaxima.push_back(i);
+          theConsecutiveStrips.push_back(numberOfConsecutiveStrips);
+        }
+      } else {  //not the case for looking for the additional maxima
+        theMaxima.push_back(i);
+        theConsecutiveStrips.push_back(numberOfConsecutiveStrips);
+      }
+    }  //if maximafound
+  }    //all pulses
+}  //find maxima procedure
 
 bool CSCHitFromStripOnly::isPeakOK(int iStrip, float heightCluster) {
   int i = iStrip;

--- a/RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc
+++ b/RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc
@@ -146,17 +146,15 @@ CSCRecHit2D CSCMake2DRecHit::hitFromStripAndWire(const CSCDetId& id,
   float ymiddle;
 
   // First wire of first wiregroup in cluster
-  wglo = layergeom_->middleWireOfGroup(wglo) + 0.5 
-    - 0.5 * layergeom_->numberOfWiresPerGroup(wglo);
-
+  wglo = layergeom_->middleWireOfGroup(wglo) + 0.5 - 0.5 * layergeom_->numberOfWiresPerGroup(wglo);
+  
   // Last wire in last wiregroup of cluster (OK if only 1 wiregroup too)
-  wghi = layergeom_->middleWireOfGroup(wghi) - 0.5 
-    + 0.5 * layergeom_->numberOfWiresPerGroup(wghi);
+  wghi = layergeom_->middleWireOfGroup(wghi) - 0.5 + 0.5 * layergeom_->numberOfWiresPerGroup(wghi);
   
   float stripWidth = -99.f;
   // If at the edge, then used 1 strip cluster only
   if (centerStrip == 1 || centerStrip == specs_->nStrips() || nStrip < 2) {
-    lp0 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wglo, wghi )).first;
+    lp0 = (layergeom_->possibleRecHitPosition(float(centerStrip) - 0.5, wglo, wghi)).first;
     ymiddle = lp0.y();
     positionWithinTheStrip = 0.f;
     stripWidth = layergeom_->stripPitch(lp0);
@@ -164,7 +162,7 @@ CSCRecHit2D CSCMake2DRecHit::hitFromStripAndWire(const CSCDetId& id,
     quality = 2;
   } else {
     // If not at the edge, used cluster of size ClusterSize:
-    LocalPoint lp11 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wglo, wghi )).first;
+    LocalPoint lp11 = (layergeom_->possibleRecHitPosition(float(centerStrip) - 0.5, wglo, wghi)).first;
     ymiddle = lp11.y();
     stripWidth = layergeom_->stripPitch(lp11);
 
@@ -181,7 +179,7 @@ CSCRecHit2D CSCMake2DRecHit::hitFromStripAndWire(const CSCDetId& id,
                                positionWithinTheStrip,
                                sigmaWithinTheStrip,
                                quality);
-    lp0 = LocalPoint( xWithinChamber, ymiddle );
+    lp0 = LocalPoint(xWithinChamber, ymiddle);
   }
 
   // compute the errors in local x and y

--- a/RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc
+++ b/RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc
@@ -156,7 +156,7 @@ CSCRecHit2D CSCMake2DRecHit::hitFromStripAndWire(const CSCDetId& id,
   float stripWidth = -99.f;
   // If at the edge, then used 1 strip cluster only
   if (centerStrip == 1 || centerStrip == specs_->nStrips() || nStrip < 2) {
-    lp0 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wilo, wihi )).first;
+    lp0 = (layergeom_->possibleRecHitPosition( float(centerStrip) - 0.5, wglo, wghi )).first;
     ymiddle = lp0.y();
     positionWithinTheStrip = 0.f;
     stripWidth = layergeom_->stripPitch(lp0);

